src/cutadapt/__main__.py

```diff
@@ -286,6 +285,10 @@ def get_argument_parser():
             "depending on input. The summary report is sent to standard output. "
             "Use '{name}' in FILE to demultiplex reads into multiple "
             "files. Default: write to standard output")
+    group.add_argument('--compression-level', default=6, dest="compresslevel",
+        help= 'Compression level if gzipped output files are used.  Default: %(default)s')
+    group.add_argument('-Z', action='store_true', dest='Z',
+        help= 'Short-hand for `--compression_level 1`. ')
```

Comment:
You can write it like this:
```
group.add_argument('-Z', action='store_const', const=1, dest='compression_level', help=...)
```
and then remove the `compresslevel = 1 if args.Z else args.compresslevel` line below.

I know the code is inconsistent right now, but can you also please use double quotes for new code? I’m considering switching to black, which will convert everything to double quotes anyway. Better to have a smaller diff then.

Reply:
I prefer double quotes as well. But I checked the other code... :stuck_out_tongue: Will change!